docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md

```diff
@@ -0,0 +1,103 @@
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+<button class="button button-orange" disabled>Open in Colab</button>
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/public/models/bert_sequence_classifier_banking77_en_3.3.2_2.4_1636277240241.zip){:.button.button-orange.button-orange-trans.arr.button-icon}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler() \
+    .setInputCol('text') \
+    .setOutputCol('document')
+
+tokenizer = Tokenizer() \
+    .setInputCols(['document']) \
+    .setOutputCol('token')
+
+sequenceClassifier = BertForSequenceClassification \
+      .pretrained('bert_sequence_classifier_banking77', 'en') \
+      .setInputCols(['token', 'document']) \
+      .setOutputCol('class') \
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['I am still waiting on my card?']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler() 
+    .setInputCol("text") 
+    .setOutputCol("document")
+
+val tokenizer = Tokenizer() 
+    .setInputCols("document") 
+    .setOutputCol("token")
+
+val tokenClassifier = BertForSequenceClassification.pretrained("bert_sequence_classifier_banking77", "en")
+      .setInputCols("document", "token")
+      .setOutputCol("class")
+      .setCaseSensitive(true)
+      .setMaxSentenceLength(512)
+
+val pipeline = new Pipeline().setStages(Array(document_assembler, tokenizer, sequenceClassifier))
+
+val example = Seq.empty["I am still waiting on my card?"].toDS.toDF("text")
+
+val result = pipeline.fit(example).transform(example)
+```
+</div>
+
+## Results
+
+```bash
+['atm_support']
+```
+
+{:.model-param}
+## Model Information
+
+{:.table-model}
+|---|---|
+|Model Name:|bert_sequence_classifier_banking77|
+|Compatibility:|Spark NLP 3.3.2+|
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/philschmid/BERT-Banking77](https://huggingface.co/philschmid/BERT-Banking77)
```

Comment:
There is a benchmark we can use: 
![image](https://user-images.githubusercontent.com/36634572/140911442-c6ab7e1e-9fb8-45c9-a70c-5bd3bdcce8a6.png)